### DISCUSSION TITLE
Lastest build for async needed

### DISCUSSION THREAD

**Discussion by frusio** [Sporif/dxvk-async#13] [Q&A] [answered]
Greetings to all developers here. I need the latest master build (i believe it named 525fd53) since it fixed some graphical glitch in the game i played. Can someone build it?

Thank you very much in advance!

**Comment by grazaerd** [comment, marked as answer]
@frusio the game is dragon nest? If so, the graphical is fdn/skill effect?

**Reply by ghost** [reply]
@frusio have a look at my discussion and you'll see the latest commit posted there yesterday 🙏

**Comment by frusio** [comment]
Sorry for late reply, i was buried at work in the last 2 weeks.

@dnblank123 No, i played Portal 2. I'm hoping DN' black pixel glitch fixed tho, its AMD's problem afaik.
@seanmacky thank you so much, man. i really need it since today i got long day off.

**Reply by grazaerd** [reply]
@frusio Yea, but I manage to fix it now. I already tested in AMD its fixed also.
